Makefile

```diff
@@ -1,9 +1,16 @@
-DOCKER      ?= docker
-HUGO_VERSION = $(shell grep ^HUGO_VERSION netlify.toml | tail -n 1 | cut -d '=' -f 2 | tr -d " \"\n")
-DOCKER_IMAGE = kubernetes-hugo
-DOCKER_RUN   = $(DOCKER) run --rm --interactive --tty --volume $(CURDIR):/src
-NODE_BIN     = node_modules/.bin
-NETLIFY_FUNC = $(NODE_BIN)/netlify-lambda
+HUGO_VERSION      = $(shell grep ^HUGO_VERSION netlify.toml | tail -n 1 | cut -d '=' -f 2 | tr -d " \"\n")
+NODE_BIN          = node_modules/.bin
+NETLIFY_FUNC      = $(NODE_BIN)/netlify-lambda
+
+# The CONTAINER_ENGINE variable is used for specifying the container engine. By default 'docker' is used
+# but this can be overridden when calling make, e.g.
+#   CONTAINER_ENGINE=podman make container-image
+CONTAINER_ENGINE ?= docker
+CONTAINER_IMAGE   = kubernetes-hugo
+CONTAINER_RUN     = $(CONTAINER_ENGINE) run --rm --interactive --tty --volume $(CURDIR):/src
+
+CCRED=\033[0;31m
+CCEND=\033[0m
```

Comment:
Assumes ANSI-style terminal, but probably OK!

Reply:
I made this assumption based on this line (which already contains a color encoding as well): https://github.com/kubernetes/website/blob/master/Makefile#L18

So I think this should be OK indeed.